src/components/common/Navbars/NavBar.js

```diff
@@ -127,16 +129,16 @@ function NavBar(props) {
               <Menu.Item key="1" icon={<HomeOutlined />}>
                 <NavLink to="/">Home</NavLink>
               </Menu.Item>
-              {!localStorage.getItem('okta-token-storage') && (
-                <Menu.Item key="3" icon={<LoginOutlined />}>
-                  <NavLink to="/login">Login</NavLink>
-                </Menu.Item>
-              )}
-              {localStorage.getItem('okta-token-storage') && (
-                <Menu.Item key="99" icon={<ProfileIcon />}>
-                  <NavLink to="/dev">My Dashboard</NavLink>
-                </Menu.Item>
-              )}
+              {/* {!localStorage.getItem({}) && ( */}
```

Comment:
You can remove all this. I don't want any references to local storage to haunt us.